packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss

```diff
@@ -74,9 +74,15 @@
 
     &:focus-within {
       .gux-tag-remove-icon {
-        border-radius: var(--gse-semantic-focusRing-cornerRadius);
-        outline: var(--gse-semantic-focusRing-small-border-width) solid
-          var(--gse-semantic-color-focus, --gse-ui-color-focus);
+        border-radius: var(
+          --gse-semantic-focusRing-cornerRadius,
+          var(--gse-semantic-focusOutline-sm-borderRadius)
+        );
+        outline: var(
+            --gse-semantic-focusRing-small-border-width,
+            var(--gse-semantic-border-focus)
```

Comment:
This is a colour fallback for a width token.

Reply:
fixed